tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -20,6 +20,7 @@ public class SurveyTool extends HttpServlet {
     static final Logger logger = SurveyLog.forClass(SurveyTool.class);
 
     private static final long serialVersionUID = 1L;
+    private static final String FAVICON_LINK = "<link rel='icon' href='/cldr-apps/favicon.ico' type='image/x-icon'>\n";
```

Comment:
```suggestion
    private static final String FAVICON_LINK = "<link rel='icon' href='./favicon.ico' type='image/x-icon'>\n";
```

or relative to context root.  We need to get rid of all static `cldr-apps` paths

Reply:
good idea!